graphrag_sdk/attribute.py

```
@@ -159,3 +174,23 @@ def __str__(self) -> str:
+    Args:
+        attributes (list[list[str]]): The attributes extracted from the graph.
+
+    Returns:
+        processed_attributes (list[Attribute]): The processed attributes.
+    """
+    processed_attributes = []
+    for attr in attributes:
+        try:
+            type = AttributeType.from_string(attr[0][1])
+            processed_attributes.append(Attribute(attr[0][0],type))
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)